src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -1067,18 +1097,30 @@ public Color(global::System.Drawing.Color color) : this(Interop.Vector4.NewVecto
         {
         }
 
-        internal Color(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        internal Color(ColorChangedCallback cb, float r, float g, float b, float a)
         {
+            callback = cb;
+            this.r = r;
+            this.g = g;
+            this.b = b;
+            this.a = a;
         }
 
-        internal Color(ColorChangedCallback cb, float r, float g, float b, float a) : this(Interop.Vector4.NewVector4(ValueCheck(r), ValueCheck(g), ValueCheck(b), ValueCheck(a)), true)
+        internal Color(ColorChangedCallback cb, Color other) : this(cb, other.R, other.G, other.B, other.A)
         {
-            callback = cb;
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
-        internal Color(ColorChangedCallback cb, Color other) : this(cb, other.R, other.G, other.B, other.A)
+        internal void CopyValuesFromDali()
```

Comment:
```suggestion
        /// <summary>
        /// This API MUST be called when we update Color values from dali by SwigCPtr
        /// </summary>
        internal void CopyValuesFromDali()
```

We need to remain some comments when we *MUST* call this API.